README.md

```diff
@@ -562,12 +598,12 @@ You may specify the base with or without a trailing slash.
 The hash-based manager uses the fragment identifier to store navigation state. This makes it a better fit for applications that are served as a static HTML file:
 
 ```ts
-import createHashHistory from '@dojo/routing/history/createHashHistory';
+import HashHistory from '@dojo/routing/history/HashHistory';
 
-const history = createHashHistory();
+const history = new HashHistory();
 ```
 
-The `history` object has the same `current` getter and `set()` and `replace()` methods. The `createHashHistory()` factory too assumes the global object is a browser `window` object, but an explicit object can be provided. It'll access `window.history` and add an event listener for the `hashchange` event.
+The `history` object has the same `current` getter and `set()` and `replace()` methods. The `HashHistory` class too assumes the global object is a browser `window` object, but an explicit object can be provided. It'll access `window.history` and add an event listener for the `hashchange` event.
```

Comment:
Not that it was your change, but "The `HashHistory` class too assumes the global object is a browser `window` object, but an explicit object can be provided." reads oddly